Registrar.Database/Stores/RegiSqlUserStore.cs

```diff
@@ -42,30 +43,22 @@ public RegiSqlUserStore(string connectionString) : base(connectionString)
             }
         }
 
-        public async Task<RegiUser> GetUserByEmail(string email)
+        public async Task<IEnumerable<RegiUser>> GetUsers()
         {
-            try
-            {
-                using (var connection = await GetConnectionAsync())
-                {
-                    var result = await connection.QueryAsync(RegistrarQueries.UserGetByEmail, new {Email = email});
+            var users = await GetUserByQuery(RegistrarQueries.UserGetAll, new {});
```

Comment:
Creating a pointless new object? Null not supported? Probably should overload the getuser method